src/helpers/fetchData.js

```diff
@@ -0,0 +1,37 @@
+const fetch = require('node-fetch');
+const { YOUTUBE_API_KEY, YOUTUBE_CHANNEL_ID } = require('../config');
+
+async function fetchLatestYoutubeVideos(options) {
+  const mrOpt = 'maxResults' in options ? `maxResults=${options.maxResults}` : ``;
+  const paOpt = 'publishedAfter' in options ? `publishedAfter=${options.publishedAfter}` : '';
+  const optUrl = [mrOpt, paOpt].join('&') || '';
```

Comment:
I don't think this will ever reach the right side. Wont this cause an issue if both values are an empty string then it will just append an `&` to the url...?

Reply:
If in the url is an `?` the following is parsed as parameters. If and & is appended to the end of the url string nothing will happen.
So if options is empty the url will be something like this `https://www.googleapis.com/youtube/v3/search?key=${YOUTUBE_API_KEY}&channelId=${YOUTUBE_CHANNEL_ID}&part=snippet&order=date&&` and the last two `&` are ignored.

Reply:
Even though it's ignored. It can be solved like this: 
```javascript
const optUrl = [mrOpt, paOpt].filter(Boolean).join('&') || '';
```

Also, don't close conversations before the author has seen your response and answered. Joseph might have something to add or comment on.

Reply:
> Also, don't close conversations before the author has seen your response and answered or reacted to it. @spiray might have something to add or comment on.
 
My bad.

> `const optUrl = [mrOpt, paOpt].filter(Boolean).join('&') || '';`

Nice, I´m adding it right now. 🙌